lib/paradocs/policies.rb

```diff
@@ -157,6 +157,10 @@ def ok?(options, actual)
       !payload.key?(key) || ok?(options, actual)
     end
 
+    meta_data do |opts|
+      {options: opts}
```

Comment:
meta_data is merged between the policies in docs metadata. the key should be unique, unless you want it to be rewritten

Reply:
I'd suggest you using `length` key and guarantee opts is hash